tests/meta/test_invoice.py

```diff
@@ -0,0 +1,111 @@
+import mock
+import pytest
+
+import cg.meta.invoice
+from tests.store_helpers import StoreHelpers
+from cg.meta.invoice import InvoiceAPI
+from tests.mocks.limsmock import MockLimsAPI
+
+from cg.store import Store, models
+from cg.constants.record_type import RecordType
+from cg.constants.priority import PriorityTerms
+
+
+def test_assert_invoice_api(
+    store: Store,
+    lims_api: MockLimsAPI,
+    helpers: StoreHelpers,
+    invoice_id: int = 0,
+    record_type: str = RecordType.Sample,
+    customer_id: str = "cust001",
+):
+    # GIVEN an invoice
+    invoice = helpers.ensure_invoice(
+        store,
+        invoice_id=invoice_id,
+        record_type=record_type,
+        customer_id=customer_id,
+    )
+    # ASSERT that an API can be initiated
+    api = InvoiceAPI(store, lims_api, invoice)
+    assert isinstance(api, InvoiceAPI)
+
+
+def test_invoice_api_sample(
+    store: Store,
+    lims_api: MockLimsAPI,
+    helpers: StoreHelpers,
+    invoice_id: int = 0,
+    record_type: str = RecordType.Sample,
+    customer_id: str = "cust132",
+):
+    """Test that the invoice records the right record_type"""
+    # GIVEN an invoice
+    invoice = helpers.ensure_invoice(
+        store,
+        invoice_id=invoice_id,
+        record_type=record_type,
+        customer_id=customer_id,
+    )
+
+    # THEN calling InvoiceAPI should return an API
+    api = InvoiceAPI(store, lims_api, invoice)
+    assert api
+    # THEN record_type should be Sample
+    assert api.record_type == record_type
+    # THEN prepare should set an invoice_info dictionary
+    api.prepare("ki")
+    assert api.invoice_info["priority"] == PriorityTerms.STANDARD
+
+
+def test_invoice_api_pool_cust032(
+    store: Store,
+    lims_api: MockLimsAPI,
+    helpers: StoreHelpers,
+    invoice_id: int = 0,
+    record_type: str = RecordType.Pool,
+    customer_id: str = "cust032",
```

Comment:
Repeatedly hard coded - use already existing customer fixture if nothing special with the value